### ISSUE TITLE
After creating a new profile property, when I am taken back to the source overview, the list of profile properties is not updated.

### ISSUE THREAD

**Issue by andyjih** [grouparoo/grouparoo#349]
Router.back() doesn't reload the page

**Comment by evantahler** [comment]
@bleonard I'm seeing newly made profile properties appear in the list on `/source/[guid]/overview`.  Also rule that are deleted disappear in the same way.  Can you add any more detail on how to reproduce this?

**Comment by andyjih** [comment]
@evantahler This happens to me, specifically when I'm not in dev mode. Could that be the difference?

I'm seeing another bug/issue that might be related:
- Source has 1 profile property rule
- Create a new profile property rule for this source
- I'm back on the Source detail page and I see 0 profile property rules
- If I refresh I now see 2 profile property rules.

**Comment by andyjih** [comment]
@evantahler confirmed - the list is complete when I'm in dev mode, but if I just do `npm run start` the list does _not_ refresh with the newly created rule.

**Comment by evantahler** [comment]
TLDR: Never use `Router.basck()` because this messes with the back button too... that's bad
